contracts/YearnGelatoOps.sol

```diff
@@ -300,11 +300,12 @@ contract YearnGelatoOps {
         require(gelatoFeeToken == feeToken, "!token"); // dev: gelato not using intended token
         require(gelatoFee <= maxFee, "!fee"); // dev: gelato executor overcharnging for the tx
 
-        // Re-run harvestTrigger() with the gelatoFee passed by the executor to ensure
-        // the tx makes economic sense.
-        require(strategy.harvestTrigger(gelatoFee), "!economic");
+        // Re-run strategyReportTrigger() to ensure the tx makes economic sense.
+        // yearn-v3: we pass the strategy address to the CommonReportTrigger to check for reportTrigger
+        (bool shouldReport, ) = commonReportTrigger.strategyReportTrigger(strategyAddress);
+        require(shouldReport, "!economic");
 
-        strategy.harvest();
+        strategy.report();
```

Comment:
is it no longer called a "harvest"? If not, I suggest we update the function/event names accordingly